pyproject.toml

```
@@ -108,6 +108,10 @@ module = [
     "lightning.app.cli.commands.connection",
     "lightning.app.cli.commands.lightning_cli",
     "lightning.app.cli.commands.cmd_install",
+    "lightning.app.cli.commands.pwd",
+    "lightning.app.cli.commands.ls",
+    "lightning.app.cli.commands.cp",
+    "lightning.app.cli.commands.cdß",
```

Accidental character there?
```suggestion
    "lightning.app.cli.commands.cd",
```

---

Good catch !